CHANGELOG.md

```diff
@@ -1,5 +1,146 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-beta.5
+
+<!-- generated comparing v5.0.0-beta.4..next -->
+
+_Aug 24, 2021_
+
+Big thanks to the 24 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐛 Fixed lots of regressions and bugs to get us closer to the [v5 stable release milestone](https://github.com/mui-org/material-ui/milestone/44)
+- 🗎 Improved the docs and the migration guide for upgrading to v5
+
+### `@material-ui/core@5.0.0-beta.5`
+
+#### Breaking changes
+
+- <!-- 36 --> [core] Update `.browserslistrc` file (#27788) @DanailH
+
+The targets of the default bundle have changed.
+
+It now supports the following minimum versions:
+
+<!-- #stable-snapshot -->
+
+- Chrome 90 (up from 84)
+- Edge 91 (up from 85)
+- Safari 14 (macOS) (up from 13.1) and 12.4 (iOS) (up from 12.2)
+- Opera 76 (up from 70)
+- and more (see [.browserslistrc (`stable` entry)](https://github.com/mui-org/material-ui/blob/HEAD/.browserslistrc#L11))
+
+If you need to support IE 11, check out our [legacy bundle](https://next.material-ui.com/guides/minimizing-bundle-size/#legacy-bundle).
```

Comment:
```suggestion
  The targets of the default bundle have changed:

  - Chrome 90 (up from 84)
  - Edge 91 (up from 85)
  - Safari 14 (macOS) (up from 13.1) and 12.4 (iOS) (up from 12.2)
  - Opera 76 (up from 70)
```

This focuses only on the change to the prior release and groups the description properly under the listitem